exchanges/okcoin/okcoin_test.go

```diff
@@ -77,1251 +80,1670 @@ func TestStart(t *testing.T) {
-	update := `{"table":"spot/depth","action":"update","data":[{"instrument_id":"BTC-USD","asks":[["3864.6786","0",0],["3864.9852","0",0],["3865.9994","0.48402971",1],["3866.4004","0.001",1],["3866.7995","0.3273",2],["3867.4566","0",0],["3867.7031","0.025",1],["3868.0436","0",0],["3868.346","0",0],["3868.3695","0.051",1],["3870.9243","0.642",1],["3874.9942","0.51751796",1],["3875.7057","0",0],["3939","0.001",1]],"bids":[["3864.55","0.0565449",1],["3863.8282","0",0],["3863.8153","0",0],["3863.7898","0.01320077",1],["3863.4807","0.02112123",1],["3863.3002","0.04233533",1],["3863.1717","0.03379397",1],["3863.0685","0.04438179",1],["3863.0286","0.7362564",1],["3862.9912","0.06773651",1],["3862.8626","0.05407035",1],["3862.7595","0.07101087",1],["3862.313","0.3756",2],["3862.1848","0.012",1],["3862.0734","0",0],["3861.8391","0.025",1],["3861.7888","0",0],["3856.6716","0.38893641",1],["3768","0",0],["3766.66","0",0],["3766","0",0],["3765.58","0",0],["3762.58","0",0],["3761","0",0],["3760.8031","0",0],["3760.7777","0",0]],"timestamp":"2019-03-06T23:19:18.239Z","checksum":-1587788848}]}`
```

Comment:
I am noting the removal of orderbook checksum tests. I would like to see them return because they are still relevant

Reply:
The order book data checksum value they send with the push data may not be correct. I tried multiple combinations following their rule but couldn't get the result.

Reply:
Even more of a reason to have tests to demonstrate that it doesn't work. Would you like some help going over it?

Reply:
I've sent you a diff.diff to apply that should get the checksum validation working. _It works on my machine_ 🤞